sql/core/src/test/scala/org/apache/spark/sql/TestData.scala

```diff
@@ -57,6 +58,15 @@ object TestData {
       TestData2(3, 2) :: Nil, 2).toDF()
   testData2.registerTempTable("testData2")
 
+  case class DoubleData(a: JavaDouble, b: JavaDouble)
```

Comment:
i think we should now just create these datasets in the parts that need them. The ones here were mostly pre-dataframe when it was pretty hard to create these.